.vscode/cspell.json

```diff
@@ -7,6 +7,7 @@
     "**/test-resources.json",
     ".devcontainer/devcontainer.json",
     ".devcontainer/oncreate",
+    ".github/CODEOWNERS",
     ".github/dependabot.yml",
     ".vscode/cspell.json",
     ".vscode/extensions.json",
```

Comment:
@weshaggard there are enough of "misspellings" in these files, should we just ignore these directories entirely?

(I wouldn't hold up this PR for it, though.)

Reply:
I've got to resolve this to merge, so I'm going to do that and we could move this conversation elsewhere. My personal thought is that it's not a big deal to exclude new files as they come for now.

Reply:
Yes I'm OK excluding the entire directory.